srcpkgs/scheme48/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'scheme48'
+pkgname=scheme48
+version=1.9
+revision=2
```

Comment:
`revision` is used to distinguish consecutive Void builds of program from same sources. Please set version to 1.9.2, revision to 1, drop `wrksrc` and adapt usages of $version.